coralproject/talk
# [iOS CSS Fix] Improve legibility of the verify text

## What does this PR do?
iOS sets disabled inputs to `opacity: 0.4`. This PR explicitly sets the `opacity` to `1` so the text is more legible.

## How do I test this PR?
- Load Talk on iOS and click Delete My Account
- Continue thru to step 3 and notice the text is now readable
